src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/onetoplevelclass/InputOneTopLevelClassAnnotation.java

```diff
@@ -0,0 +1,11 @@
+package com.puppycrawl.tools.checkstyle.checks.design.onetoplevelclass;
+
+@interface InputOneTopLevelClassAnnotation {
+   String author();
+   String date();
+   int currentRevision() default 1;
+   String lastModified() default "N/A";
+   String lastModifiedBy() default "N/A";
+   // Note use of array
+   String[] reviewers();
```

Comment:
I see no need for that many annotation variables

Reply:
Reduced to 1 or 2 fields per definition